torchdata/datapipes/iter/load/online.py

```diff
@@ -84,9 +84,10 @@ class GDriveReaderDataPipe(IterDataPipe[Tuple[str, StreamWrapper]]):
     Args:
         source_datapipe: a DataPipe that contains URLs to GDrive files
     """
+    source_datapipe: IterDataPipe[str]
 
     def __init__(self, source_datapipe: IterDataPipe[str]) -> None:
-        self.source_datapip: IterDataPipe[str] = source_datapipe
```

Comment:
Lol

Reply:
It has an error because an 'e' is missing?

Reply:
Yeah